DataStructures/Lists/MergeSortedArrayList.java

```diff
@@ -0,0 +1,60 @@
+package DataStructures.Lists;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author https://github.com/shellhub
+ */
+
+public class MergeSortedArrayList {
+    public static void main(String[] args) {
+        List<Integer> listA = new ArrayList<Integer>();
+        List<Integer> listB = new ArrayList<Integer>();
+        List<Integer> listC = new ArrayList<Integer>();
```

Comment:
Use `new ArrayList<>()` instead of `new ArrayList<Integer>()`

Reply:
I have updated. Thanks for your review. @yanglbme